common/flags/Flags.java

```diff
@@ -104,7 +104,13 @@ static void setFlagValue(String name, String value) {
 
   @VisibleForTesting
   static String getDefaultFlagValue(String flagName) {
-    return instance().getFlag(flagName).getDefault();
+    FlagData flagData = instance().getFlag(flagName);
+    if(flagData.getIsListFlag()){
```

Comment:
Please format as:
`if (flagData.getIsListFlag()) {`
(note the spaces)
Also elsewhere.